pyproject.toml

```diff
@@ -68,7 +69,7 @@ include = ["volatility3*"]
 mypy_path = "./stubs"
 show_traceback = true
 
-[tool.mypy.overrides]
```

Comment:
What are the double brackets here for?

Reply:
`ignore_missing_imports` for mypy is actually supposed to be only for specific modules (and it needs to be wrapped in double brackets). There's a warning if it isn't an array:
```
pyproject.toml: tool.mypy.overrides sections must be an array. Please make sure you are using double brackets like so: [[tool.mypy.overrides]]
```

Reply:
But seeing as how there's no specific module specified in `[[tool.mypy.overrides]]`, should I just remove it entirely since it's not serving any purpose right now?

Reply:
Errr, I think I'd prefer it.  I might come along and think the double brackets are a mistake otherwise...  Either remove it or find some way to comment it with the documentation that says why double bracketing makes sense (I'm still not sure I understand why the section itself needs double bracketing, I think it's being interpreted strangely in some way, I'd just as soon deal with it later if it's not adding anything/necessary).